webhook-bridge/svix-webhook-bridge.example5.senders.yaml

```diff
@@ -0,0 +1,44 @@
+log_level: "debug"
+log_format: "json"
+
+# Optionally configure these settings to export OTEL data
+opentelemetry:
+  address: "http://localhost:1234"
+  service_name: "my-bridge"
+  sample_ratio: 0.5
+
+# Host/port to listen on for HTTP inputs.
+# Only required when HTTP receivers are specified, probably?
+# http_listen_address: "0.0.0.0:5000"
+
+
+# Each defines an `input`, (optional) `transformation`, and `output`.
+senders:
+  - name: "from-rabbit-local-to-svix"
+    input:
+      type: "rabbitmq"
+      queue: "local"
+      uri: "amqp://example.com/%2f"
+    # Optional transformation. JSON bodies will be forwarded as-is when unset.
```

Comment:
Omitted the `requeue_on_nack` --  let's assume we can pick a default either way and document what it is, though this feels like a "stopped clock is correct twice a day" situation to me.